istio/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,57 @@
+name: Istio
+files:
+  - name: istio.yaml
+    options:
+    - template: init_config
+      options:
+        - template: init_config/openmetrics
+        - template: init_config/default
+    - template: instances
+      options:
+        - name: istio_mesh_endpoint
+          description: |
+            To enable Istio metrics you must specify the url exposing the API.
+            Note for RHEL and SUSE users: due to compatibility issues, the check does not make use of
+            the CPP extension to process Protocol buffer messages coming from the api. Depending
+            on the metrics volume, the check may run very slowly.
+          value:
+            default: null
+            example: http://istio-telemetry.istio-system:42422/metrics
+            type: string
+        - name: mixer_endpoint
+          description: |
+            Define the mixer endpoint in order to collect all Prometheus metrics on the Mixer process as well
+            as gRPC metrics related to API calls and metrics on adapter dispatch.
+            If using Istio < v1.1, replace port 15014 with port 9093. See the changes here:
+            https://istio.io/about/notes/1.1/#configuration-management
+          value:
+            default: null
+            example: http://istio-telemetry.istio-system:15014/metrics
+            type: string
+        - name: pilot_endpoint
+          description: |
+            Define the pilot endpoint in order to collect all Prometheus metrics on the Pilot process.
+            Only available for Istio >= v1.1.
+          value:
+            default: null
+            example: http://istio-pilot.istio-system:15014/metrics
+            type: string
+        - name: galley_endpoint
+          description: |
+            Define the galley endpoint in order to collect all Prometheus metrics on the Galley process.
+            Only available for Istio >= v1.1.
+          value:
+            default: null
+            example: http://istio-galley.istio-system:15014/metrics
+            type: string
+        - name: citadel_endpoint
+          description: |
+            Define the citadel endpoint in order to collect all Prometheus metrics on the Citadel process.
+            Only available for Istio >= v1.1.
+          value:
+            default: null
+            example: http://istio-citadel.istio-system:15014/metrics
+            type: string
+        - template: instances/openmetrics
+          overrides:
+            prometheus_url.hidden: true
```

Comment:
```suggestion
            prometheus_url.hidden: true
        - template: instances/default
```

Reply:
the `instances/default` template was included in the openmetrics instance template. Should I remove it from the openmetrics template for consistency or add `init_config/default` template to `init_config/openmetrics`

Reply:
the latter